lua/lspconfig/clangd.lua

```diff
@@ -12,7 +12,27 @@ local function switch_source_header(bufnr)
   end)
 end
 
+-- https://clangd.llvm.org/extensions.html#file-status
+local function file_status_update(_, _, message, client_id)
+  local client = vim.lsp.get_client_by_id(client_id)
+  if not client then
+    vim.api.nvim_err_writeln("LSP[" .. client_id .. "] client has shut down after sending the message")
+    vim.api.nvim_command("redraw")
```

Comment:
Why is there a redraw here?

Reply:
This is how `err_message` is defined in handlers.lua in core.